python/fledge/services/core/scheduler/scheduler.py

```diff
@@ -303,6 +311,16 @@ async def _start_task(self, schedule: _ScheduleRow) -> None:
         args_to_exec.append("--address=127.0.0.1")
         args_to_exec.append("--name={}".format(schedule.name))
 
+        # Check a service / task token exists or create a new one
+        startToken = self._startupTokens.get(schedule.name, None)
+        if startToken is None:
+            letters = string.ascii_lowercase
+            startToken = ''.join((random.choice(string.ascii_lowercase) for x in range(32)))
+            self._startupTokens[schedule.name] = startToken
```

Comment:
We should ask core service registry to assign a single use token and then use that to check and allow registration?